src/main/java/games/strategy/triplea/util/BonusIncomeUtils.java

```diff
@@ -8,26 +8,26 @@
 import games.strategy.util.IntegerMap;
 
 
-public class AiBonusIncomeUtils {
+public class BonusIncomeUtils {
 
   /**
    * Add AI bonus income based on the player's set percentage for all resources.
```

Comment:
Yeah, javadoc needs updated here. I'll do that in the next PR around these changes since its minor and still changing.